### ISSUE TITLE
[Deeplab]: Pretrained checkpoints provided lack 1,005 operations

### ISSUE THREAD

**Issue by hoonkai** [tensorflow/models#6767]
### Describe the problem

If we examine [xception65_coco_voc_trainaug](http://download.tensorflow.org/models/deeplabv3_pascal_train_aug_2018_01_04.tar.gz), it contains 2,339 operations. However, if we train using [xception_65_imagenet](http://download.tensorflow.org/models/deeplabv3_xception_2018_01_04.tar.gz) for 10 steps with
```
python deeplab/train.py \
    --logtostderr \
    --training_number_of_steps=10 \
    --train_split="train" \
    --model_variant="xception_65" \
    --atrous_rates=6 \
    --atrous_rates=12 \
    --atrous_rates=18 \
    --output_stride=16 \
    --decoder_output_stride=4 \
    --train_crop_size=513 \
    --train_crop_size=513 \
    --train_batch_size=1 \
    --dataset="pascal_voc_seg" \
    --tf_initial_checkpoint=${PATH_TO_INITIAL_CHECKPOINT} \
    --train_logdir=${PATH_TO_TRAIN_DIR} \
    --dataset_dir=${PATH_TO_DATASET}
```

and freeze the graph with

```
freeze_graph \
--input_graph=graph.pbtxt \
--input_checkpoint=model.ckpt-10\
--input_binary=false \
--output_graph=frozen.pb \
--output_node_names=ResizeBilinear_3
```

the frozen graph contains just 1,647 operations. However, it contains 313 operations not common with xception65_coco_voc_trainaug. Same result for [xception_65_imagenet_coco](http://download.tensorflow.org/models/xception_65_coco_pretrained_2018_10_02.tar.gz). One difference appears to be the way batch normalisation is done. How come the models trained on VOC (and Cityscapes) have over 1,000 new operations? Can the 'updated' pretrained models be made available?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by hoonkai** [comment]
> What is the top-level directory of the model you are using

deeplab

> Have I written custom code

No.

> OS Platform and Distribution

Ubuntu 16.04

> TensorFlow installed from

source

> TensorFlow version

r1.13

> Bazel version
0.22

> CUDA/cuDNN version

9.1

> GPU model and memory

n/a
> Exact command to reproduce

As above.

**Comment by TianMingChen** [comment]
> Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
> What is the top-level directory of the model you are using
> Have I written custom code
> OS Platform and Distribution
> TensorFlow installed from
> TensorFlow version
> Bazel version
> CUDA/cuDNN version
> GPU model and memory
> Exact command to reproduce

Can you upload the train DELF code?
